lib/portus/registry_client.rb

```diff
@@ -26,13 +26,13 @@ def initialize(host, use_ssl = false, username = nil, password = nil)
     # Returns whether the registry is reachable with the given credentials or
     # not.
     def reachable?
-      res = perform_request("", "get", false)
+      res = perform_request("/v2/", "get", false)
 
       # If a 401 was retrieved, it means that at least the registry has been
       # contacted. In order to get a 200, this registry should be created and
       # an authorization requested. The former can be inconvenient, because we
       # might want to test whether the registry is reachable.
-      !res.nil? && res.code.to_i == 401
+      !res.nil? && res.code.to_i == 401 || res.code.to_i == 200
```

Comment:
`(res.code.to_i == 401 || res.code.to_i == 200)`, right?

Reply:
it should work, however its more clear as you suggested.

http://ruby-doc.org/core-2.4.2/doc/syntax/precedence_rdoc.html